.github/workflows/test-run-server-sdk-e2e-tests.yml

```
@@ -0,0 +1,18 @@
+name: Test server SDK E2E tests
+on:
+  pull_request:
```

```suggestion
  pull_request:
```
```suggestion
  workflow_dispatch:
```

Let's trigger manually to not run tests on each PR to the DX team toolkit